<launch>
  <!-- Fake laser -->
  <node pkg="icp" type="icp" name="icp_node" output="screen" launch-prefix="xterm -e gdb --args"/>
</launch>
